bitcoin/src/psbt/map/input.rs

```diff
@@ -207,12 +216,16 @@ impl PsbtSighashType {
     ///
     /// Allows construction of a non-standard or non-valid sighash flag
     /// ([`EcdsaSighashType`], [`TapSighashType`] respectively).
-    pub fn from_u32(n: u32) -> PsbtSighashType { PsbtSighashType { inner: n } }
+    pub fn from_u32(n: u32) -> PsbtSighashType {
+        PsbtSighashType { inner: n }
+    }
```

Comment:
In 7df9b3d76d9388639a4d6f208de8a98a50fdf5a9:

Can you drop these formatting changes? Our weekly fmt job is just going to revert them.

Reply:
Done, `rust-analyzer` rug-pulled me with the autoformat.

Reply:
I posted elsewhere but this is likely caused by using `rustfmt` with `stable` instead of `nightly`.